crates/core/tedge/src/cli/config/config_key.rs

```diff
@@ -51,6 +51,7 @@ impl ConfigKey {
             config_key!(AzureMapperTimestamp),
             config_key!(MqttBindAddressSetting),
             config_key!(MqttPortSetting),
+            config_key!(HttpPortSetting),
```

Comment:
You can also add `HttpBindAddressSetting`, making this setting read only as for `DeviceIdSetting`.

Reply:
Done

Reply:
Why read-only? We don't want to let users configure the `HttpBindAddress`? If not, why are we even adding it?

Reply:
@albinsuresh and @reubenmiller 

> Why read-only? We don't want to let users configure the `HttpBindAddress`? If not, why are we even adding it?

To be aligned with https://github.com/thin-edge/thin-edge.io/issues/1479 where we conclude that for now we will use the MQTT bind address for both HTTP and MQTT.

We just need to be consistent here. I see 3 options:

* either hide `HttpBindAddress` to the user (i.e not listed by `tedge config list`).
* or show `HttpBindAddress` but read only,  to let the user know that this is using the MQTT setting .
* or make `HttpBindAddress` an independent setting the user can list,get and set.